apps/public/src/app/bijlagen/_components/file-upload.spec.tsx

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from '@testing-library/react'
+
+import '@testing-library/jest-dom'
+import { FileUpload } from './file-upload'
+
+describe('FileUpload Component', () => {
+  it('renders the file upload button', () => {
+    render(<FileUpload />)
+    const buttonElement = screen.getByText(/Selecteer bestanden/i)
```

Comment:
[`getByRole` has preference](https://testing-library.com/docs/queries/about/#priority). I think it might fail now though, because of the button in the label.

Reply:
Couldn't find an accessible element with role.